Code/GraphMol/Fingerprints/catch_tests.cpp

```diff
@@ -549,9 +549,9 @@ TEST_CASE("AtomPairGenerator bit info", "[fpgenerator][AP]") {
   SECTION("folded bitInfo") {
     {
       AdditionalOutput::bitInfoMapType expected = {
-          {351, {{0, 1}}},
```

Comment:
It appears that this changes the output, do we need a release note?

Reply:
The title of the bug will be in the release notes, so I just made it a bit more descriptive.